gnuplot/plan.sh

```diff
@@ -0,0 +1,47 @@
+pkg_name=gnuplot
+pkg_origin=core
+pkg_version="5.0.6"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('gnuplot')
+pkg_source="https://sourceforge.net/projects/$pkg_name/files/$pkg_name/$pkg_version/$pkg_name-$pkg_version.tar.gz"
+pkg_shasum="5bbe4713e555c2e103b7d4ffd45fca69551fff09cf5c3f9cb17428aaacc9b460"
+pkg_deps=(
+  core/cairo
+  core/expat
+  core/fontconfig
+  core/freetype
+  core/gcc-libs
+  core/glib
+  core/glibc
+  core/harfbuzz
+  core/libcerf
+  core/liberation-fonts-ttf
+  core/libiconv
+  core/libgd
+  core/libpng
+  core/lua
```

Comment:
I found that `core/lua` seems to be useless in runtime deps, I see configure searches for it, but it does not change the build outcome apparently... Might be worth removing or moving it to build deps?